pages/admin/index.tsx

```diff
@@ -266,13 +349,32 @@ export default function Requests() {
                   <InputLeftElement pointerEvents="none">
                     <SearchIcon color="text.filler" />
                   </InputLeftElement>
-                  <Input placeholder="Search by first name, last name or user ID" />
+                  <Input
+                    placeholder="Search by name or user ID"
+                    onKeyDown={event => {
+                      if (event.key === 'Enter') {
+                        setSearchFilter(searchInput);
+                        setPageNumber(0);
+                      }
+                    }}
+                    onChange={event => setSearchInput(event.target.value)}
```

Comment:
Oustan and I added a [useDebounce](https://usehooks.com/useDebounce/) hook in my [latest commit](https://github.com/uwblueprint/richmond-centre-for-disability/pull/42/commits/94abb4044c7b17244d7adf3e8532ce6983f8a9af) so that we only need to use onChange here. The debouncer will only set the searchFilter state if it's value hasn't been changed in the last 500ms. That way we can avoid making users press enter to search without firing a db query every time a key is pressed. It's better ux and looks cooler imo

Can you copy those changes in here?